calico_cni_k8s_test.go

```diff
@@ -30,6 +32,8 @@ import (
 func init() {
 	// Create a random seed
 	rand.Seed(time.Now().UTC().UnixNano())
+	logrus.SetFormatter(&logutils.Formatter{})
```

Comment:
I thought you're also switching out `"log"` with logrus in this PR. That's what I meant by that comment :), it's ok if you want to do it in a separate PR if there's a bunch more files with `"log"` import